pennylane/ops/op_math/controlled.py

```diff
@@ -143,15 +143,6 @@ class Controlled(SymbolicOp):
         do_queue(bool):  indicates whether the operator should be
             recorded when created in a tape context
 
-    .. note::
-        This class, ``Controlled``, denotes a controlled version of any individual operation.
-        :class:`~.ControlledOp` adds :class:`~.Operation` specific methods and properties to the
-        more general ``Controlled`` class.
```

Comment:
Sorry about this. This section should probably stay. The stuff below paragraph about `ControlledOperation` can go.